Formula/butterfly.rb

```diff
@@ -1,8 +1,10 @@
 class Butterfly < Formula
+  VERSION = "2.3.0"
+
   desc "Application code transformation tool"
   homepage "https://github.com/paypal/butterfly"
-  url "https://search.maven.org/remotecontent?filepath=com/paypal/butterfly/butterfly-cli-package/2.2.0/butterfly-cli-package-2.2.0.zip"
-  sha256 "7ae80466c5df8ef6d2c1a55bac3b584c94ce746eee1a4cd2674b458c2cae2d16"
+  url "https://search.maven.org/remotecontent?filepath=com/paypal/butterfly/butterfly-cli-package/#{VERSION}/butterfly-cli-package-#{VERSION}.zip"
+  sha256 "1806cb4605902c31329d94bb4d87becac95b1f7108156f3864b267f48a1d0105"
```

Comment:
Hey @mmcrockett , how is this SHA generated? Also, besides this change, did you have to do any change anywhere else?

Reply:
Just searched 'brew formula get new sha' and this stack overflow was first answer - https://stackoverflow.com/questions/32673943/how-to-update-homebrew-sha256

No other changes.